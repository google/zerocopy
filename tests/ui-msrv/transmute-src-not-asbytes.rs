../ui-nightly/transmute-src-not-asbytes.rs