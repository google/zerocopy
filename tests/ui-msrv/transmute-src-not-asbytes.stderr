error[E0277]: the trait bound `NotZerocopy<AU16>: AsBytes` is not satisfied
  --> tests/ui-msrv/transmute-src-not-asbytes.rs:18:32
   |
18 | const SRC_NOT_AS_BYTES: AU16 = transmute!(NotZerocopy(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `AsBytes` is not implemented for `NotZerocopy<AU16>`
   |
note: required by a bound in `AssertIsAsBytes`
  --> tests/ui-msrv/transmute-src-not-asbytes.rs:18:32
   |
18 | const SRC_NOT_AS_BYTES: AU16 = transmute!(NotZerocopy(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertIsAsBytes`
   = note: this error originates in the macro `transmute` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `NotZerocopy<AU16>: AsBytes` is not satisfied
  --> tests/ui-msrv/transmute-src-not-asbytes.rs:18:32
   |
18 | const SRC_NOT_AS_BYTES: AU16 = transmute!(NotZerocopy(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `AsBytes` is not implemented for `NotZerocopy<AU16>`
   |
note: required by a bound in `AssertIsAsBytes`
  --> tests/ui-msrv/transmute-src-not-asbytes.rs:18:32
   |
18 | const SRC_NOT_AS_BYTES: AU16 = transmute!(NotZerocopy(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertIsAsBytes`
   = note: this error originates in the macro `transmute` (in Nightly builds, run with -Z macro-backtrace for more info)
